src/components/contactForm.tsx

```diff
@@ -82,41 +92,44 @@ export const ContactForm = () => {
           Thanks for joining!
         </Text>
       ) : (
-        <form onSubmit={state.succeeded ? emailSubmitted : trySubmit}>
-          <Stack className={styles.textMaxWidth} verticalAlignment="center" horizontalAlignment="center" grow>
-            <Input
-              value={email}
-              onChange={handleUpdateEmail}
-              danger={emailError}
-              label="Email"
-              placeholder="Enter you email"
-              type="email"
-              name="email"
-              input={{ 'aria-describedby': helperTextId }}
-            />
-            <Text className={styles.textMaxWidth} size={500} align="center">
-              or{' '}
-            </Text>
-            <Input
-              value={ig}
-              onChange={handleUpdateIg}
-              danger={emailError}
-              label="Instagram"
-              placeholder="Instagram handle"
-              type="text"
-              name="instagram"
-              input={{ 'aria-describedby': helperTextId }}
-            />
-            <Button appearance="primary" color="brand" type="submit" disabled={state.submitting}>
-              {state.succeeded ? 'Thanks!' : state.submitting ? 'Sending...' : 'Submit'}
-            </Button>
-          </Stack>
-          <div style={{ opacity: !emailError ? 0 : 1 }}>
-            <Body color="danger" id={helperTextId}>
-              Please enter a valid email
-            </Body>
-          </div>
-        </form>
+        <>
```

Comment:
I like the icons, I would remove the (optional) and keep the or, its easier to understand.

Reply:
It ends up looking strange on mobile when it collapses when using "or". Having "optional" in the label is a fix for this.

Reply:
For some more context: 
![image](https://user-images.githubusercontent.com/7192338/161638563-b43c8c01-406d-4255-9fc0-27c222a4cf2d.png)
![image](https://user-images.githubusercontent.com/7192338/161638585-4a37a71f-bef0-48ae-868e-1e9e2c8ae3a7.png)

Reply:
Pick one